test/form/samples/system-multiple-export-bindings/_expected.js

```diff
@@ -0,0 +1,37 @@
+System.register([], function (exports) {
+	'use strict';
+	return {
+		execute: function () {
+
+			const x = exports('x', 123);
+			const y = exports('y', 456);
+
+			var namespace = /*#__PURE__*/Object.freeze({
+				__proto__: null,
+				x: x,
+				y: y
+			});
+			exports({namespace: namespace, namespace2: namespace});
+
+			// Namespace variable
+
+			// Variable Declaration
+			let a = exports('default', exports('a2', exports('a', 1))), b = exports('b', 2), c = exports('c2', exports('c' = 3));
+
+			// Export default expression
+
+			// Assignment Expression
+			a = exports('default', exports('a2', exports('a', b = exports('b', c = exports('c2', exports('c', 0))))));
+
+			// Destructing Assignment Expression
+			(function (v) {exports({a: a, a2: a, default: a, b: b, c: c, c2: c}); return v;} ({a, b, c} = { c: 4, b: 5, a: 6 }));
```

Comment:
@guybedford I noticed we were already writing IIFEs for de-structured arguments, and wondered if we should be consistent in using inline exports call expressions or IIFEs

Reply:
The IIFE is the preferable form whenever possible. Note that this works because the binding updates apply after computing the RHS, including any default expressions there which could themselves have side effects. As a result there is no inbetween phase (as far as I'm aware) where some bindings are set and others aren't - so the IIFE doing them all in one is always preferable when we can.